src/main/java/com/epam/ta/reportportal/core/analyzer/auto/impl/SearchLogServiceImpl.java

```diff
@@ -177,7 +178,14 @@ private SearchLogRs composeResponse(Launch launch, Log log, Map<Long, String> pa
 		response.setDuration(log.getTestItem().getItemResults().getDuration());
 		response.setStatus(log.getTestItem().getItemResults().getStatus().name());
 		response.setIssue(IssueConverter.TO_MODEL.apply(log.getTestItem().getItemResults().getIssue()));
-		response.setLogMessages(Arrays.asList(log.getLogMessage()));
+		response.setLogs(Lists.newArrayList(TO_LOG_ENTRY.apply(log)));
 		return response;
 	}
+
+	private static final Function<Log, SearchLogRs.LogEntry> TO_LOG_ENTRY = log -> {
```

Comment:
Maybe it has sense to move it to LogConverter and make it public?

Reply:
done